frontend/packages/dev-console/src/components/topology/__tests__/topology-test-data.ts

```diff
@@ -1449,6 +1450,96 @@ export const samplePipelineRun: FirehoseResult = {
   ],
 };
 
+export const serviceBindingRequest = {
```

Comment:
nit: this file has already grown, shall we have in new as in `__mocks__/service-binding-mock.ts`

Reply:
Done.